### PR TITLE
Feature/angular init

### PR DESCRIPTION
**Pull request by davey-gov**
Initial setup, still needs loads of work

### CODE REVIEW

**Comment by scottqueen-bixal** [comment]
Presentation Issues:

- Showing Pending Title, but title is "asf"
- Showing "Pending Title" as H2 in the details of the accordion, should be "Procurement Title"

Screen Shot 2021-11-01 at 3 00 12 PM

- Office Agency Name and Funding Agency Name should have case handling, can you confirm if we are handling this on the data script rather than the front end?
Screen Shot 2021-11-01 at 3 00 22 PM

- Several fields have no data, previously we fell back to "N/A"
Screen Shot 2021-11-01 at 3 00 29 PM

- Only one accordion should be open at a time
Screen Shot 2021-11-01 at 3 00 52 PM

- padding is incorrect around accordion button
Screen Shot 2021-11-01 at 3 19 12 PM

- container border should match accordion button border
Screen Shot 2021-11-01 at 3 21 57 PM

- fields should stack at tablet-portrait width
(should be similar to this....)

screencapture-madeinamerica-gov-waivers-2021-11-01-15_23_19

Console Errors:

> ERROR Error: Uncaught (in promise): Error: Cannot match any routes. URL Segment: 'preview/gsa/made-in-america/feature/angular-init/waivers'
Error: Cannot match any routes. URL Segment: 'preview/gsa/made-in-america/feature/angular-init/waivers'

causes errors on reload of application in stagged sites....

https://user-images.githubusercontent.com/37077057/139727970-5c52b5ea-fead-4eb0-80e5-258d1b63b5d2.mov

> Refused to apply style from 'https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/waivers/uswds-styles.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

> main-4c1c5bc54e73a1c…520f46f14311ad.js:1 Service worker registration failed with: TypeError: Failed to register a ServiceWorker for scope ('https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/waivers/') with script ('https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/waivers/ngsw-worker.js'): A bad HTTP response code (404) was received when fetching the script.

Don't think we need service workers here, let's make sure it is turned off?

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -34,6 +34,9 @@ jobs:
 
       - name: Install NPM dependencies.
         run: npm install
+
```
Comment (scottqueen-bixal) [inline .github/workflows/build-and-test.yml:37]: zomie code?

File: search-app/src/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>UsaComponents</title>
```
Comment (scottqueen-bixal) [inline search-app/src/index.html:5]:
```suggestion
    <title>Non-Availability Waivers</title>
```

**Comment by scottqueen-bixal** [comment]
- Filtering from 'All' to, 'Reviewed', and back to 'All' does not list _All_ the waivers
- Filtering does not include the option for "Submitted", this input should probably be dynamically built out based on all the status options available in the array - so that if there is another status made available in the data we don't have to statically add it in.
- "Did the contracting Agency compete this opportunity?"  has no "N/A" fall back for empty fields
- Selected pagination should be white even when not focused, currently failing WCAG 2 AA contrast ratio thresholds - the state should look similar to https://designsystem.digital.gov/components/pagination/
- missing arrows on paggination component
> Failed to load resource: the server responded with a status of 404 () /assets/img/sprite.svg#navigate_before:1

**Comment by clmedders** [comment]
Page numbers at the bottom of the page are showing black when clicked 

Number - bottom of screen

**Comment by clmedders** [comment]
Frist waiver T700 turboshaft Engine - under request status, submitted is lower case I'm guessing this is a content import issue

**Comment by clmedders** [comment]
Should the question "did the contacting agency compete this opportunity?" be left blank?

**Comment by clmedders** [comment]
After you choose alphabetical and go from page 2 back to page 1 it is no longer alphabetical

https://user-images.githubusercontent.com/88721460/140073215-38e3e535-5357-4b65-9df8-0ee0fecf3db7.mp4

.

**Comment by clmedders** [comment]
Rayon waiver is repeated on the page

**Comment by clmedders** [comment]
When the waiver is opened the title is repeated
title is repeated

**Comment by clmedders** [comment]
when I switch to alphabetical the last waiver in the list on page 2 starts with "P" should be moved up in the list.

**Comment by clmedders** [comment]
should there be a submitted filter status?

**Comment by clmedders** [comment]
when I am filtering by status sometimes it will populate and not show the page number at the bottom and just call all the waivers.

**Comment by scottqueen-bixal** [comment]
> When the waiver is opened the title is repeated title is repeated

I think this is by design

**Comment by scottqueen-bixal** [comment]
> when I am filtering by status sometimes it will populate and not show the page number at the bottom and just call all the waivers.

can you provide a screen capture here for this issue, if there is less than 11 items in the array we won't have pagination

**Comment by clmedders** [comment]
> > when I am filtering by status sometimes it will populate and not show the page number at the bottom and just call all the waivers.
> 
> can you provide a screen capture here for this issue, if there is less than 11 items in the array we won't have pagination

Yes, sorry forgot to add that one in... I did a screen record thought that might be the best.  When I am first at all it has 10 and then 1 on the next page then when I go from assigned then back to all under status it has all 11 on the page.

https://user-images.githubusercontent.com/88721460/140081049-87aa7ff5-ea2c-4cee-af2a-0de8e5877444.mp4

**Pull request merged**